docs/syntax/metrics.md

```diff
@@ -7,117 +7,43 @@ permalink: /docs/syntax/metrics/
 
 <table class="o-techdocs-table">
 <tr>
-	<th>Property</th>
-	<th>Type</th>
+	<th>envName</th>
 	<th>Description</th>
+	<th>Example variation</th>
 </tr><tr>
-	<td><code>{</code></td>
-	<td></td>
-	<td></td>
-</tr><tr>
-	<td>&nbsp;&nbsp;<code>schemaVersion</code></td>
-	<td>number*</td>
-	<td>Set to 1.</td>
-</tr><tr>
-	<td>&nbsp;&nbsp;<code>metrics&nbsp;{</code></td>
-	<td>object*</td>
-	<td>A list of metrics</td>
-</tr><tr>
-	<td>&nbsp;&nbsp;&nbsp;&nbsp;<code><em>metricname</em>&nbsp;{</code></td>
-	<td>object*</td>
-	<td>An object representing a single metric.  The object's key is the name of the metric, and may be defined by the service.</td>
-</tr><tr>
-	<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<code>type</code></td>
-	<td>string*</td>
-	<td>
-		The type of value.  One of:
-		<ul>
-		<li><code>counter</code>: The value is a number representing a count, e.g. number of running processes, items in a queue, bytes of disk space remaining, seconds since last new content, etc.</li>
-		<li><code>boolean</code>: The value is a simple boolean true/false, e.g. whether the service can connect to its database.</li>
-		<li><code>movingaverage</code>: The value is an average of a set of multiple values over a period of time, e.g. the response time of a depended-upon service.</li>
-		</ul>
-	</td>
-</tr><tr>
-	<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<code>val</code></td>
-	<td><em>varies</em>*</td>
-	<td>
-		The current value of the metric.  In the case of counter metrics, a number.  In the case of boolean metrics, a boolean.  Should not be present for movingaverage metrics.
-	</td>
-</tr><tr>
-	<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<code>description</code></td>
-	<td><em>string</em></td>
-	<td>
-		Optional short description to provide any supplemental information about this metric
-	</td>
-</tr><tr>
-	<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<code>unit</code></td>
-	<td><em>string</em>*</td>
-	<td>Count only.  Plural name of the unit of measurement in which val is expressed.  Common values should be 'seconds', 'bytes', 'items', 'processes'.  Prefer base level orders of magnitude (ie. express time in seconds, not milliseconds, and data size in bytes, not megabytes)</td>
+	<td><code>dev</code></td>
+	<td>Development.  An environment directly edited by a developer, often on a local machine, rarely stable, may be spun up and down intermittently. The developer name is a good way of separating these</td>
+	<td><code>dev-andrew</code></td>
 </tr><tr>
-	<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<code>since</code></td>
-	<td><em>string</em></td>
-	<td>Count only.  ISO8601 format date specifying the time at which the count was last reset.</td>
+	<td><code>qa</code></td>
```

Comment:
We're in danger of creating a standard for how the org names its lower environments!